### DISCUSSION TITLE
Phone confirmation on change

### DISCUSSION THREAD

**Discussion by bombillazo** [supabase/supabase#14875] [Questions] [answered]
Hello, similar to how updating a user's email triggers sending a confirmation email, how can one send a confirmation SMS to the new phone to confirm that the change is valid?

**Comment by j4w8n** [comment, marked as answer]
This might be what you're looking for: https://supabase.com/docs/reference/javascript/auth-resend

Although the "resend" wording implies that one should've already been sent.
